shimming-toolbox/shimmingtoolbox/shim/sequencer.py

```diff
@@ -820,29 +821,29 @@ def calc_shimmed_anat_orient(self, coefs, list_shim_slice):
         nib.save(nii_shimmed_anat_orient, fname_shimmed_anat_orient)
 
     def _plot_static_signal_recovery_mask(self, unshimmed, shimmed_Gz, mask):
-    # Plot signal loss maps
-        def calculate_signal_loss(B0_map):
-            G = np.gradient(B0_map, axis = 2)
-            signal_map = abs(np.sinc(self.epi_te * G))
+        # Plot signal loss maps
+        def calculate_signal_loss(gradient):
+            slice_thickness = self.nii_anat.header['pixdim'][3]
```

Comment:
Does this include slice gap?

Reply:
no, you're right I looked at an acquisition of 1mm with a slice gap of 1mm and pixdim[3] is 2

Reply:
I don't see any parameter that gives the correct value though

Reply:
The JSON file should include a parameter that has slice gap information.